<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab5Class</class>
 <widget class="QMainWindow" name="Lab5Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab5</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="outHeadWidget" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>5</y>
       <width>241</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="headLayout">
      <item>
       <widget class="QLabel" name="output">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Result:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLineEdit" name="input1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Lab5.qrc"/>
 </resources>
 <connections/>
</ui>
